bucket/detect-it-easy.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "3.01",
+    "description": "Program for determining types of files.",
+    "homepage": "http://ntinfo.biz",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/horsicq/DIE-engine/releases/download/3.01/die_win64_portable_3.01.zip",
+            "hash": "65B7731D23C34CEE5D2EDBDC0D02CAF2D9363BE41CE1D1C2B0D23534B99EBD10"
+        },
+        "32bit": {
+            "url": "https://github.com/horsicq/DIE-engine/releases/download/3.01/die_win32_portable_3.01.zip",
+            "hash": "4274DC088543216F6305414BC1B46D56EA51E6A952CB657684753755541DC8B6"
+        }
+    },
+    "bin": [ "die.exe", "diec.exe" ],
+    "shortcuts": [
+        ["die.exe", "Detect It Easy"]
+    ],
+    "persist": "die.ini",
+    "checkver": {
```

Comment:
Project does not use GitHub releases (https://github.com/horsicq/Detect-It-Easy/releases)